src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session_ComputeModel.cs

```diff
@@ -103,119 +101,44 @@ public void Do()
 
                         // TODO(cyrusn): We're calling into extensions, we need to make ourselves resilient
                         // to the extension crashing.
-                        var groups = await GetGroupsAsync(_completionService, _triggerInfo, cancellationToken).ConfigureAwait(false);
-                        if (groups == null)
+                        var completionList = await GetCompletionListAsync(_completionService, _triggerInfo, cancellationToken).ConfigureAwait(false);
+                        if (completionList == null)
                         {
                             return null;
                         }
 
-                        groups.Do(AddGroupToMap);
-                        if (_displayNameToItemsMap.Count == 0)
-                        {
-                            return null;
-                        }
-
-                        var totalItems = _displayNameToItemsMap.Values.Flatten().ToList();
-                        totalItems.Sort();
-
                         var trackingSpan = await _completionService.GetDefaultTrackingSpanAsync(_documentOpt, _subjectBufferCaretPosition, cancellationToken).ConfigureAwait(false);
                         return Model.CreateModel(
                             _disconnectedBufferGraph,
                             trackingSpan,
-                            totalItems,
-                            selectedItem: totalItems.First(),
+                            completionList.Items.ToList(),
+                            selectedItem: completionList.Items.First(),
                             isHardSelection: false,
                             isUnique: false,
                             useSuggestionCompletionMode: _includeBuilder,
```

Comment:
For my own edification.  If we have a builder, then we'll show 'suggestion mode' right?  This is just to allow us to *always* show suggestion mode if that's what the user prefers?